src/components/Select.js

```diff
@@ -38,6 +41,11 @@ const Select = React.createClass({
     };
   },
 
+  componentDidMount () {
+    styles.option[':hover'].backgroundColor = this.props.color || StyleConstants.Colors.PRIMARY;
+    styles.activeItem.backgroundColor = this.props.color || StyleConstants.Colors.PRIMARY;
```

Comment:
rather than setting the fallback color here, you can set the default prop to `StyleConstants.Colors.PRIMARY`.